internal/ncproxy/store/store.go

```diff
@@ -0,0 +1,279 @@
+package store
+
+import (
+	"context"
+	"encoding/json"
+
+	ncproxynetworking "github.com/Microsoft/hcsshim/internal/ncproxy/networking"
+	"github.com/pkg/errors"
+	bolt "go.etcd.io/bbolt"
+)
+
+var (
+	ErrBucketNotFound = errors.New("bucket not found")
+	errKeyNotFound    = errors.New("key does not exist")
+)
+
+type NetworkingStore struct {
```

Comment:
Can this struct and `ComputeAgentStore` be moved into their own submodules to reduce redundancy in the names? 
ie:
```go
import "github.com/Microsoft/hcsshim/internal/ncproxy/store/network"

type service struct {
    s  network.Store
}

func f() {
    svc := &service {
        s: network.NewStore(...)
    }
}
```

Reply:
Hmm. One concern with doing that is that the name `network` is not very descriptive and could get confusing with the `network` variables for hcn and ncproxy. Thoughts?

Reply:
oh yeah, that would overshadow all those variables.
description wise, I think fairly short names are common for internal packages (especially in containerd).
If you think itll hurt readability, then Im fine keeping it as is.